uf3/regression/least_squares.py

```diff
@@ -346,10 +348,10 @@ def combine_weighted_gram(self,
             gram (np.ndarray): gram matrix (x^T x) for fitting.
             ordinate (np.ndarray): ordinate (x^T y) for fitting.
         """
-        gram = ((weight * energy_weight**2 * gram_e)
-                + ((1 - weight) * force_weight**2 * gram_f))
-        ordinate = ((weight * energy_weight**2 * ord_e)
-                    + ((1 - weight) * force_weight**2 * ord_f))
+        gram = (((weight * energy_weight) ** 2 * gram_e)
```

Comment:
should be reverted to have `**2` only on `energy_weight`.

(following 3 lines as well)